<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetLocationDetails</class>
 <widget class="QWidget" name="WidgetLocationDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  color: white;
  /*background-color: rgb(50,82,95);*/
}
QGroupBox {
  border-radius: 0px;
  border: 3px solid rgb(131, 147, 167);
  background-color: rgb(80,112,125);
}
QLineEdit {
  background-color: rgb(101,117, 137);  
}
QPushButton {
  background: transparent;
}
QPushButton:hover {
  background-color: white;
  border: 1px solid rgb(115, 210, 22);
  border-radius:  4px;
}
QPushButton[enabled=&quot;false&quot;] {
  background-color: rgb(31,47,67);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1">
            <item>
             <widget class="QLabel" name="labelDescription">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
  border: 1 px white;
  border-radius: 2px;
}</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButtonPhoto">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
  background-color: transparent;
  border: 0;
}
QPushButton:hover {
  background-color: transparent;
  border: 0;
}</string>
              </property>
              <property name="icon">
               <iconset resource="../resources/application.qrc">
                <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelLine">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
  background-color: rgb(131,147, 167);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Voting members:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayoutVotes">
              <property name="spacing">
               <number>8</number>
              </property>
              <item>
               <widget class="QWidget" name="widgetVotes" native="true">
                <property name="toolTip">
                 <string>No votes</string>
                </property>
                <property name="styleSheet">
                 <string notr="true">
  border-radius: 2px;
  border: 1px solid rgb(131, 147, 167);
  background-color: rgb(131,147, 167);
</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="leftMargin">
                  <number>6</number>
                 </property>
                 <property name="topMargin">
                  <number>3</number>
                 </property>
                 <property name="rightMargin">
                  <number>6</number>
                 </property>
                 <property name="bottomMargin">
                  <number>3</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="font">
                    <font>
                     <family>Roboto</family>
                     <pointsize>9</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(238, 238, 236);</string>
                   </property>
                   <property name="text">
                    <string>Total</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelVotes">
                   <property name="font">
                    <font>
                     <family>Roboto</family>
                     <pointsize>9</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(238, 238, 236);</string>
                   </property>
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="labelVoteMembers">
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/application.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>onBtnVoteDownClicked()</slot>
  <slot>onBtnVoteUpClicked()</slot>
  <slot>onBtnSettingsClicked()</slot>
 </slots>
</ui>
